index.html

```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```

Comment:
So, now I am trying to cretae a suggestion in the code, using the "blue plus icon"
```suggestion
<h2>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```